examples/react/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2024 web3js-react-dapp-demo
```

Comment:
Is it important to change this?

Reply:
I removed example app license as I don't think it is relevant here, I kept plugin license in the repo's root